actors/market/src/lib.rs

```diff
@@ -1022,65 +1017,66 @@ impl Actor {
     }
 }
 
-fn compute_data_commitment<BS: Blockstore>(
+fn deal_proposals<BS: Blockstore>(
```

Comment:
Nit: `load_proposals` or `get_proposals` or similar; the word "deal" isn't adding any meaning.